src/main/java/uk/gov/companieshouse/web/pps/controller/pps/BankTransferPenaltyReferenceController.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.companieshouse.web.pps.controller.pps;
+
+import jakarta.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.view.UrlBasedViewResolver;
+import uk.gov.companieshouse.web.pps.annotation.PreviousController;
+import uk.gov.companieshouse.web.pps.controller.BaseController;
+import uk.gov.companieshouse.web.pps.models.AvailablePenaltyReference;
+import uk.gov.companieshouse.web.pps.models.PenaltyReferenceChoice;
+import uk.gov.companieshouse.web.pps.util.PenaltyReference;
+
+@Controller
+@PreviousController(PPSStartController.class)
+@RequestMapping("/late-filing-penalty/bank-transfer/which-penalty-service")
+public class BankTransferPenaltyReferenceController extends BaseController {
+    private static final String PPS_BANK_TRANSFER_PENALTY_REFERENCE = "pps/bankTransferPenaltyReference";
+
+    private static final String PPS_BANK_TRANSFER_LATE_FILING_PATH = "/late-filing-penalty/bank-transfer/late-filing-details";
+    private static final String PPS_BANK_TRANSFER_SANCTIONS_PATH = "/late-filing-penalty/bank-transfer/sanctions-details";
```

Comment:
Sonar is saying to get these from a customisable parameter. Should we be getting these from a properties file @dgordon-ch?

Reply:
In theory yes, anything we're using more than once especially. We might be able to leave this for a separate cleanup/debt task to remove that smell.

Reply:
Sounds good.